Makefile

```diff
@@ -5,22 +5,22 @@ ifndef HAVE_RULES
     HAVE_RULES=no
 endif
 
+# Find available Python interpreter
```

Comment:
hmm.. I can envision this could cause problems for somebody who builds cppcheck using mingw and doesn't have python.

Reply:
That's possible. I wasn't sure if there a case where no Python would be available at all. Not having Python would not be very useful though since you cannot use the matchcompiler or run all tests.

I will try if the detection can be put in a target which is run conditionally.

Reply:
I was not able to get it to work inside a target without duplicating the code. For the inline detection I used a shorter form though.

I also added some TODOs for potential issues I came across during testing.

Reply:
ok :+1: